modules/microshift-lvms-csi-snapshot-optional.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-lvms-csi-snapshot-optional_{context}"]
+= LVMS CSI provider and CSI snapshot deployments
+
+{microshift-short} can now be configured to disable the built-in logical volume manager storage (LVMS) Container Storage Interface (CSI) provider or the CSI snapshot capabilities. These capabilities can reduce the use of runtime resources such as RAM, CPU, and storage.
```

Comment:
```suggestion
You can disable the built-in logical volume manager storage (LVMS) Container Storage Interface (CSI) provider or the CSI snapshot capabilities. These capabilities can reduce the use of runtime resources such as RAM, CPU, and storage.
```
"These capabilities can reduce the use of runtime resources such as RAM, CPU, and storage." Doesn't make sense. Do you mean removing the storage component reduces the use of runtime storage?

Reply:
disabling the CSI provider or the CSI snapshot capabilities can reduce use of runtime resources.

Sentence can be rewritten as 
You can disable the built-in logical volume manager storage (LVMS) Container Storage Interface (CSI) provider or the CSI snapshot capabilities to reduce the use of runtime resources such as RAM, CPU, and storage.

Reply:
Removing the CSI entirely versus disabling snapshotting are very different. Which is it? @copejon could you clarify?

Reply:
@aravipra Do we have to do both of these things, or just one? What's the use case(s)? Like, we would we do one or the other in production? I think a little "why" information around use cases would be helpful.

Reply:
MicroShift allows for disabling one or both of the CSI components (driver or snapshotter). Primarily this is to help save cpu/mem runtime overhead. It also gives users the option to install their own CSI driver should they want to, but this is unsupported.

Reply:
have made a change.